fog/view/server/src/db_fetcher.rs

```diff
@@ -351,6 +363,8 @@ impl<DB: RecoveryDb + Clone + Send + Sync + 'static> DbFetcherThread<DB> {
                     );
                     // We might have more work to do, we aren't sure because of the error
                     may_have_more_work = true;
+                    // Let's back off for one interval when there is an error
+                    sleep(DB_POLL_INTERNAL);
```

Comment:
I am not sure if this is a good idea, but right now I think nothing backs off if we hit the error path, which might be a bad thing.